digitalpy/core/digipy_configuration/action_key.py

```diff
@@ -1,4 +1,4 @@
-from digitalpy.config.action_key_provider import ActionKeyProvider
+from digitalpy.core.digipy_configuration.action_key_provider import ActionKeyProvider
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module